farm/infer.py

```diff
@@ -251,6 +251,11 @@ def inference_from_dicts(self, dicts, rest_api_schema=False, max_processes=128):
                 f"b) ... run inference on a downstream task: make sure your model path {self.name} contains a saved prediction head"
             )
 
+        # Assign a sequential doc_id to each dictionary
+        if rest_api_schema:
```

Comment:
As I understood, we are now using baske_id instead. So we probably don't need this one anymore?